src/components/HotkeySettingDialog.vue

```diff
@@ -387,14 +425,24 @@ const resetHotkey = async (action: string) => {
   > :deep(.scroll) {
     overflow-y: scroll;
     overflow-x: hidden;
+
+    > .q-table {
+      max-width: 50rem;
+    }
   }
 
   tbody tr {
-    td button:last-child {
+    cursor: pointer;
+
+    &:hover td::before {
+      // Quasarデフォルトの強調表示を流用し、hover中の行を強調表示
+      content: "";
+    }
```

Comment:
ワークアラウンドであることはコードからわからないので、メモを残しておいてあげるとリファクタリングの役に立つかもです！
こんな感じとかどうでしょう。
```suggestion
    &:hover td::before {
      // Quasarデフォルトの強調表示が何故か働かないのでワークアラウンド
      // Quasarデフォルトの強調表示を流用し、hover中の行を強調表示
      content: "";
    }
```